### PR TITLE
SSIS fixes

### PR DESCRIPTION
**Pull request by traceon**
Aiming at solving issues from:
https://github.com/ClickHouse/clickhouse-odbc/issues/176
https://github.com/ClickHouse/clickhouse-odbc/issues/202

### CODE REVIEW

**Comment by hooman72** [comment]
This is great!
i hope it gets done fast

File: driver/utils/utils.h [hunk 1]
```diff
@@ -141,6 +142,60 @@ struct UTF8CaseInsensitiveCompare {
     }
 };
 
+inline auto parseCatalogFnVLArgs(const std::string & value_list) {
```
Comment (Enmk) [inline driver/utils/utils.h]: I would love to see at least some comments on what this function does and why.

**Comment by traceon** [comment]
> I see that following public methods were modified and I wasn't able to find any tests verifying those modifications

I don't think there were any tests for the original functions, hence no tests for the modifications.

**Comment by Enmk** [comment]
> > I see that following public methods were modified and I wasn't able to find any tests verifying those modifications
> 
> I don't think there were any tests for the original functions, hence no tests for the modifications.

Then this is a perfect opportunity to add some.

**Comment by traceon** [comment]
We need to test that (just as lots of other untested pieces we have in the driver code right now), but the time, required for implementing a proper testing for these specifically, could turn out to be quite long.

**Comment by hooman72** [comment]
I love tests and everything but can I somehow get a beta version?
right now nothing is working for me so having something is better than nothing.

**Pull request merged**